packages/flatpak.rb

```diff
@@ -96,5 +96,9 @@ def self.postinstall
     puts 'Configuring flathub'.lightblue
     system 'flatpak.elf remote-add --if-not-exists --user flathub https://flathub.org/repo/flathub.flatpakrepo'
     puts
+    xdgconfig_in_bashrc = `grep -c "#{CREW_PREFIX}/var/lib/flatpak/exports/share" ~/.bashrc || true`
+    unless xdgconfig_in_bashrc.to_i.positive?
+      system "sed -i 's,XDG_DATA_DIRS=.*,XDG_DATA_DIRS=#{CREW_PREFIX}/share:#{HOME}/.local/share/flatpak/exports/share:#{CREW_PREFIX}/var/lib/flatpak/exports/share,g' ~/.bashrc"
```

Comment:
Should we check if `XDG_DATA_DIRS=...` exists in `~/.bashrc` also?  It's possible that `xdg_base` is installed but the environment variables haven't been added.

Reply:
Good call.

Reply:
Made changes as requested.